modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -370,4 +370,68 @@ def query_params(url)
       expect(response).to have_http_status(:unprocessable_entity)
     end
   end
+
+  context 'mobile flag' do
+    it 'responds with a boolean mobile flag for VHA facilities' do
+      create :vha_648A4
+      get base_query_path + '/vha_648A4', params: nil, headers: accept_json
+      expect(response).to be_successful
+      expect(response.body).to be_a(String)
+      json = JSON.parse(response.body)
+      expect(json['data']['attributes']['mobile']).to eq(false)
+      expect(json['data']['attributes']['mobile']).to_not be_nil
+    end
+
+    it 'responds with null mobile flag for non-VHA facilities' do
+      create :nca_907
+      create :vba_348
+      create :vc_0617V
+
+      get base_query_path + lat_long, params: nil, headers: accept_json
+
+      expect(response).to be_successful
+      expect(response.body).to be_a(String)
+
+      json = JSON.parse(response.body)
+      nca = json['data'][0]
+      vba = json['data'][1]
+      vc = json['data'][2]
+
+      expect(nca['mobile']).to be_nil
+      expect(vba['mobile']).to be_nil
+      expect(vc['mobile']).to be_nil
+    end
+  end
+
+  context 'active_status flag' do
```

Comment:
When describing a context, start its description with "when" or "with".

```suggestion
  context 'with active_status flag' do
```